From b74517d77ad499162ebb235268d6ea85f27c86ab Mon Sep 17 00:00:00 2001
From: Daniel Vetter <daniel.vetter@ffwll.ch>
Date: Tue, 16 Jul 2013 09:11:56 +0200
Subject: drm/gem: remove drm_gem_object_handle_unreference

It's unused, everyone is using the _unlocked variant only.

Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Reviewed-by: Rob Clark <robdclark@gmail.com>
Signed-off-by: Dave Airlie <airlied@gmail.com>
(cherry picked from commit da5cbe361c55fe17ef94d2587991997f81f8c4cb)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 include/drm/drmP.h | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/include/drm/drmP.h b/include/drm/drmP.h
index ea284972a33c..a3e10cde0e6f 100644
--- a/include/drm/drmP.h
+++ b/include/drm/drmP.h
@@ -1605,24 +1605,6 @@ drm_gem_object_handle_reference(struct drm_gem_object *obj)
 }
 
 static inline void
-drm_gem_object_handle_unreference(struct drm_gem_object *obj)
-{
-	if (obj == NULL)
-		return;
-
-	if (atomic_read(&obj->handle_count) == 0)
-		return;
-	/*
-	 * Must bump handle count first as this may be the last
-	 * ref, in which case the object would disappear before we
-	 * checked for a name
-	 */
-	if (atomic_dec_and_test(&obj->handle_count))
-		drm_gem_object_handle_free(obj);
-	drm_gem_object_unreference(obj);
-}
-
-static inline void
 drm_gem_object_handle_unreference_unlocked(struct drm_gem_object *obj)
 {
 	if (obj == NULL)
-- 
1.8.5.rc3

